tests/unit/converter/test_biostudies_converter.py

```diff
@@ -84,7 +84,12 @@ def __get_expected_payload(hca_project):
                     },
                     {
                         "type": "Author",
+                        "accno": "a1",
                         "attributes": [
+                            {
+                                "name": "Name",
+                                "value": contributors[0]['name']
+                            },
                             {
                                 "name": "First Name",
                                 "value": _parse_name(contributors[0]['name'], 0)
```

Comment:
This is generating test data with the production code.
What is the purpose of the test?

Reply:
? What kind of production code are you talking?